modules/apps/29-fee/ibc_middleware.go

```diff
@@ -168,6 +168,14 @@ func (im IBCMiddleware) OnChanCloseConfirm(
 		return err
 	}
 
+	if im.keeper.IsLocked(ctx) {
+		return types.ErrFeeModuleLocked
+	}
+
+	if !im.keeper.IsFeeEnabled(ctx, portID, channelID) {
```

Comment:
Can you update the tests also please? :-)

Reply:
Yes ! I updated test.